src/xrCore/LocatorAPI.cpp

```diff
@@ -1934,6 +1931,29 @@ bool CLocatorAPI::can_modify_file(pcstr fname)
     }
 }
 
+bool CLocatorAPI::folder_exist(pcstr fullpath) // only Win ?
```

Comment:
Maybe use std::filesystem::exists ?

Reply:
Looks like there will be problems with the files in DB-archives.